src/options.ts

```diff
@@ -194,7 +195,27 @@ const inferOptions = (inputs: Inputs): Options => {
     inputs.pixiBinPath
   )
   const logLevel = inputs.logLevel ?? (core.isDebug() ? 'vv' : 'default')
-  const manifestPath = inputs.manifestPath ? path.resolve(untildify(inputs.manifestPath)) : 'pixi.toml'
+  // infer manifest path from inputs or default to pixi.toml or pyproject.toml depending on what is present in the repo.
+  let manifestPath = 'pixi.toml' // default
+  if (inputs.manifestPath) {
+    manifestPath = path.resolve(untildify(inputs.manifestPath))
+  } else {
+    if (existsSync('pixi.toml')) {
+      manifestPath = 'pixi.toml'
+    } else if (existsSync('pyproject.toml')) {
+      const fileContent = readFileSync('pyproject.toml', 'utf-8')
+      const parsedContent = parse(fileContent)
+      // only use pyproject.toml if [tool.pixi] is present
+      if (parsedContent.tool && parsedContent.tool.pixi) {
+        core.info('The tool.pixi table found so using pyproject.toml as manifest file.')
```

Comment:
Let's make this `core.debug` as it looks a bit weird in the logs.

![image](https://github.com/prefix-dev/setup-pixi/assets/29506042/c08f4a55-21ed-4c1a-9c6c-443d31aed510)